include/deal.II/lac/precondition_block.h

```diff
@@ -458,11 +458,6 @@ class PreconditionBlockJacobi : public virtual Subscriptor,
     const_iterator &operator++ ();
 
     /**
-     * Postfix increment.
-     */
-    const_iterator &operator++ (int);
```

Comment:
same here

Reply:
I spent some time looking over this class and I think that we should just deprecate this iterator class. The returned values don't take into account the relaxation parameter (I would argue that they should, since the relaxation operator is defined as `omega * `/A(i, i)`) and none of the other relaxation block classes offer iterators at all.

We don't have any tests that I can find for this iterator class either.

Reply:
Yes, that seems sensible. I can't think of a reason why this iterator class exists...